arcus_orientation.md

```diff
@@ -25,33 +25,23 @@ script: https://kit.fontawesome.com/83b2343bd4.js
 script: https://cdn.jsdelivr.net/npm/mermaid@9.4.3/dist/mermaid.min.js
 -->
 
-## Arcus Lab Orientation
+# Arcus Computational Labs
 
 <div class = "overview">
 
 ## Overview
 
-@comment
-
-### Is this module right for me?
-
 @long_description
 
-### Details
-
-**Estimated time to completion**: @estimated_time
-
-**Pre-requisites**: It's helpful to have reviewed the Arcus website at https://arcus.chop.edu (available only on the CHOP network), to understand Arcus overall goals.
-
-**Learning Objectives**:
+### Quick Start Guide
 
-@learning_objectives
+Just want to jump into how to get an Arcus Lab, or using one you already have? Go to [our quick start guide](arcus_quickstart.md). 
```

Comment:
```suggestion
Just want to jump into how to get an Arcus Lab, or using one you already have? Go to [our quick start guide](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_quickstart.md#1). 
```

Note that currently the link goes to the raw markdown file and you want it to go to the liascript.  The link I put in won't work YET because it goes to main, but just swap "main" for "ml-quickstart" in the URL to see current state.